tests/tests_config.yml

```diff
@@ -0,0 +1,59 @@
+---
+- name: Test cockpit_* role options
+  hosts: all
+  roles:
+    - role: linux-system-roles.cockpit
+      vars:
+        cockpit_packages: minimal
+        cockpit_enabled: yes
+        cockpit_started: no
+        cockpit_config:
+          WebService:
+            LoginTitle: "hello world"
+          Session:
+            IdleTimeout: 60
+
+  tasks:
+    - meta: flush_handlers
+
+    - name: test - socket is enabled  # noqa 303 command-instead-of-module
+      command: systemctl is-enabled {{ __cockpit_daemon }}
+      changed_when: false
+
+    - name: test - socket is not active  # noqa 303 command-instead-of-module
+      command: systemctl is-active {{ __cockpit_daemon }}
+      register: result
+      failed_when: result.rc == 0
```

Comment:
https://docs.ansible.com/ansible/2.9/user_guide/playbooks_tests.html#task-results
`failed_when: result is succeeded`